src/app/people/Components/TabCard.tsx

```
@@ -0,0 +1,47 @@
+// Copyright 2019-2025 @polkassembly/polkassembly authors & contributors
+// This software may be modified and distributed under the terms
+// of the Apache-2.0 license. See the LICENSE file for details.
+
+'use client'
```

ESLint

[error] Insert `;`

(prettier/prettier)